src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -736,6 +740,167 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                 .EndControl());
         }
 
+        private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Automation_GetError()
+        {
+            yield return new FormatViewDefinition("GetErrorInstance",
+                CustomControl.Create()
+                    .GroupByProperty("PSErrorIdentifier", label: "ErrorIdentifier")
+                    .StartEntry()
+                        .AddScriptBlockExpressionBinding(@"
+                            $maxDepth = 10
```

Comment:
Do you think this will be configurable eventually?
`$maxDepth = $ErrorDisplayDepth -as [int] ? [int]$ErrorDisplayDepth : 10`

Reply:
If users hit this limit, I think we can expose it as configurable.  The only concern here is if it's possible that have a circular Error object that would recurse forever.  I don't think it's actually possible, but added this here just in case.  Hitting a depth of 10 would make a visual nightmare anyways.